provisioning/ansible_inventory_local

```diff
@@ -0,0 +1,10 @@
+pique.nikhef.nl
+tietar.nikhef.nl
```

Comment:
As provisioning tietar/vpn does not work with `ansible_local`, I might just as well throw these out of the ansible_inventory.